demo/src/app/components/dropdown/demos/basic/dropdown-basic.html

```diff
@@ -0,0 +1,8 @@
+ <div ngbDropdown [open]="true">
+    <ngbd-button class="btn btn-primary" id="dropdownMenu1" ngbDropdownToggle>Toggle dropdown</ngbd-button>
+    <div class="dropdown-menu" aria-labelledby="dropdownMenu1">
+   	    <a class="dropdown-item">Action - 1</a>
+        <a class="dropdown-item">Another Action</a>
+        <a class="dropdown-item">Something else is here</a>
+    </div>
+</div>
```

Comment:
Newline at end of file.  You should probably consider setting this option in your IDE so you don't have to think about it anymore.